pkg/agent/sysadvisor/plugin/inference/models/borwein/inferencesvc/inference_svc.pb.go

```diff
@@ -361,7 +361,9 @@ type InferenceResult struct {
 	// It depends on the inference target.
 	Output float32 `protobuf:"fixed32,3,opt,name=output,proto3" json:"output,omitempty"`
 	// threshold to judge overload or underload. should be float between 0 and 1.
-	Percentile           float32  `protobuf:"fixed32,4,opt,name=percentile,proto3" json:"percentile,omitempty"`
+	Percentile float32 `protobuf:"fixed32,4,opt,name=percentile,proto3" json:"percentile,omitempty"`
+	// model_version to identify where does the output come from.
+	ModelVersion         string   `protobuf:"bytes,5,opt,name=model_version,json=modelVersion,proto3" json:"model_version,omitempty"`
```

Comment:
what if the server side is still old version and don't provide this field

Reply:
In that case, the field will be empty, and no influence.